tutorials/EnvironmentCreation/tutorial2_adding_game_logic.py

```diff
@@ -104,7 +104,7 @@ def step(self, actions):
         return observations, rewards, terminations, truncations, infos
 
     def render(self):
-        grid = np.zeros((7, 7))
+        grid = np.full((7,7), ' ', dtype='U1')
```

Comment:
What does the second argument to np.full do here? And I googled around to see what exactly dtype U1 is but I'm confused about how this site says https://note.nkmk.me/en/python-numpy-dtype-astype/ the dtype should be <U1?

Reply:
The render space doesn't have to be the same as the observation space to be fair, but I can see how if we're overriding an array of integers with characters as the input that's a little sloppy. Granted, numpy lets you do that so it's like, ultimately it doesn't matter and is arguably cleaner code, but not as strictly correct. Could do it as a list of lists of size 49 or some other data type as well, but this is probably cleaner

Reply:
Are you sure it doesn't matter? It doesn't seem to want to write down a char in a array of floats

![Screenshot 2023-07-22 at 09 12 35](https://github.com/Farama-Foundation/PettingZoo/assets/70576987/cf6d0e66-4abd-403b-9fd3-69f46ca2eac0)

Reply:
And 'U1' is a native type in numpy that supports char
See what happens without it : 
![Screenshot 2023-07-22 at 09 14 04](https://github.com/Farama-Foundation/PettingZoo/assets/70576987/854de17f-14da-449c-9867-cfb7a1867514)

Reply:
But you're right that numpy does type inference and selects an appropriate type, so there's no need to manually specify that it's Unicode, I can remove `U1`

Reply:
Ah I see, thanks for testing it out. Looks good just doing full with a space character